virttest/vt_utils/block_disk.py

```diff
@@ -0,0 +1,393 @@
+                   if it's None
+    :type fstype: string
+    :param options: mount options, which should be separated by ","
+                    between each options
+    :type options: string
+
+    :return: True if mounted, else return False
+    :rtype: boolean
+    """
+    mount_str = "%s %s %s" % (src, dst, fstype)
+    mount_str = mount_str.replace('None', '').strip()
+    mount_list_cmd = 'cat /proc/mounts'
+
+    mount_result = process.run(mount_list_cmd, shell=True).stdout_text
+
+    for result in mount_result.splitlines():
+        if mount_str in result:
+            if options:
+                options = options.split(",")
+                options_result = result.split()[3].split(",")
+                for op in options:
+                    if op not in options_result:
+                        return False
+            return True
+    return False
+
+
+def mount(src, dst, fstype=None, options=None):
+    """
+    Mount src under dst. Even if it has been mounted already,
+    remout will be done with options.
+
+    :param src: source device or directory
+    :type src: string
+    :param dst: mountpoint
+    :type dst: string
+    :param fstype: filesystem type
+    :type fstype: string
+    :param options: mount options, which should be separated by ","
+                    between each options
+    :type options: string
+
+    :return: if mounted successfully, return True. Otherwise, return False
+    :rtype: boolean
+    """
+    options = '' if options is None else options
+    if is_mount(src, dst, fstype, options):
+        if 'remount' not in options:
+            options = 'remount,%s' % options
+    cmd = ['mount']
+    if fstype:
+        cmd.extend(['-t', fstype])
+    if options:
+        cmd.extend(['-o', options])
+    cmd.extend([src, dst])
+    cmd = ' '.join(cmd)
+    return process.system(cmd) == 0
+
+
+def umount(src, dst, fstype=None):
+    """
+    Umount src from dst, if src really mounted under dst.
+
+    :param src: source device or directory
+    :type src: string
+    :param dst: mountpoint
+    :type dst: string
+    :param fstype: filesystem type
+    :type fstype: string
+
+    :return: if umount successfully, return True. Otherwise, return False
+    :rtype: boolean
+    """
+    mounted = is_mount(src, dst, fstype)
+    if mounted:
+        from virttest import utils_package
+        package = "psmisc"
+        # check package is available, if not try installing it
+        if not utils_package.package_install(package):
+            raise RuntimeError("%s is not available/installed for fuser",
+                               package)
+        fuser_cmd = "fuser -km %s" % dst
+        umount_cmd = "umount %s" % dst
+        process.system(fuser_cmd, ignore_status=True, verbose=True, shell=True)
+        return process.system(umount_cmd, ignore_status=True, verbose=True) == 0
+    return True
+
+
+def get_linux_disks(partition=False):
```

Comment:
```suggestion
def get_disks(partition=False):
```
I remembered that this model does not support the Windows OS in this version and I think the VT framework will not cover running on the Windows OS host in a short time.
So I suggest that we remove the `_linux` namespace for the interfaces. like the [cpu](https://github.com/avocado-framework/avocado-vt/blob/master/virttest/vt_utils/cpu.py) and [sys_time](https://github.com/avocado-framework/avocado-vt/blob/master/virttest/vt_utils/sys_time.py) module to keep the consistency for the whole `vt_utils`.
CC @zhencliu @luckyh